fluent-langneg/README.md

```diff
@@ -0,0 +1,34 @@
+# fluent-langneg
+
+`fluent-langneg` is an API for negotiating languages. It's part of
+Project Fluent, a localization framework designed to unleash 
+the expressive power of the natural language.
+
+## Installation
+
+`fluent-langneg` can be used both on the client-side and the server-side.
+You can install it from the npm registry or use it as a standalone script.
+
+    npm install fluent-langneg
+
+
+## How to use
+
+```javascript
+import negotiateLanguages from 'fluent-langneg';
+
+const supported = negotiateLanguages(requested, available, default);
```

Comment:
those are locales. We will negotiate between the language part of those locales so we will negotiate languages, but we pass locales and the returned ids are locales as well. I know it's complex :(